### ISSUE TITLE
Suggest image descriptions if missing/inacurrate

### ISSUE THREAD

**Issue by Kalytis** [mastodon/mastodon#27659]
### Pitch

Offer the possibility for people to suggest image descriptions to authors of posts, if the description is missing or could be improved, and the author can accept it and it becomes the image alt text. Users can set to accept or refuse suggestions of creations, and suggestions of improvements.

### Motivation

Since the big migrations from Twitter, image descriptions seem to have decreased greatly, resulting in a less enjoyable experience for people who use alt text.

Some users are mentioning authors of posts with pre-made alt text, asking them to edit the image accordingly.

This would allow image descriptions to be added more smoothly, resulting in more images be accessible to those who need them.

The main limitation would be that it could lead to people less taking time to set alt text for their images, figuring that other people will suggest them some. Setting alt text should always remain the author's responsibility, not the community itself.

**Comment by miblodelcarpio** [comment]
Just want to express my support for this idea, both specifically and for the general principle.

For alt text, I actually reckon the mentioned limitation may have a self-correcting effect:

1. Person A posts image without alt text.
2. Persons B, C and D suggest alt text.
3. Person A expends time and effort fielding these suggestions, and realises they could've nipped this in the bud by applying alt text in the first place.

For the general principle of "readers making suggestions" – which I only mention by way of supporting the idea for alt text, so please ignore if you think it detracts from the idea – I'd welcome this for:

* Missing / Outdated URLs for things referred to
* Typos
* Translations, if we had support for authoring the same post in different languages (overriding auto-translate)

**Comment by hllizi** [comment]
I also very much support this idea. Regarding the mentioned limitation, I am not even certain that one should see it as mainly the author's responsibility to provide a description. I have sometimes provided descriptions in the mentions of posts that lacked them and found it awkward because it felt so passive-aggressive while I was just thinking, yeah, why not, might as well do that myself.

**Comment by OddDev** [comment]
Duplicate of https://github.com/mastodon/mastodon/issues/26464
